__test__/cypress/fixtures/CSV-Headers-test.csv

```diff
@@ -0,0 +1,3 @@
+first,last_name,company_name,address,city,county,state,ZipOrPost,Cell_Phone,phone,email
+James,Baggins,"Benton, John B Jr",6649 N Blue Gum St,New Orleans,Orleans,LA,70116,504-621-8927,504-845-1427,jbaggs@gmail.com
```

Comment:
Just a note -- i'd recommend, even/especially for tests to use `@example.com` domain -- someone has that email and example.com is reserved for such purposes 
https://www.rfc-editor.org/rfc/rfc2606.html#section-3